code/obj/sec_tape.dm

```diff
@@ -0,0 +1,82 @@
+/obj/sec_tape
+	anchored = 1
+	icon = 'icons/obj/sec_tape.dmi'
+	icon_state = "sec_tape_s"
+	name = "security cordon"
+	real_name = "security cordon"
+	desc = "A small cordon of security tape, used to keep assistants off crime scenes."
+	density = 1
+	flags = FPRINT | USEDELAY | ON_BORDER
+	object_flags = HAS_DIRECTIONAL_BLOCKING
+	dir = SOUTH
+
+	New()
+		..()
+		layerify()
+		tape_UpdateIcon()
+
+	proc/layerify()
```

Comment:
I'd probably put the code from this proc into update_icon instead. No need to have two procs which you're gonna always want to call together.

Reply:
Done, layerify is gone and into update icon now